Src/Base/AMReX_GpuDevice.cpp

```diff
@@ -278,7 +278,7 @@ Device::Initialize ()
     // is only available starting from CUDA 10.0, so we will
     // leave num_devices_used as 0 for older CUDA toolkits.
 
-#if AMREX_NVCC_MAJOR_VERSION >= 10
+#if (__CUDACC_VER_MAJOR__ >= 10)
```

Comment:
Are we sure this works in all compilers we work with, or do we need to guard this with an #ifdef?

Reply:
Oh nevermind, we only get here inside AMREX_USE_CUDA, so I think it's fine.